cluster/common.sh

```diff
@@ -682,6 +682,21 @@ EOF
 INITIAL_ETCD_CLUSTER: $(yaml-quote ${INITIAL_ETCD_CLUSTER})
 EOF
     fi
+    if [ -n "${APISERVER_FEATURE_CONFIG:-}" ]; then
+      cat >>$file <<EOF
+APISERVER_FEATURE_CONFIG: $(yaml-quote ${APISERVER_FEATURE_CONFIG})
```

Comment:
This is pretty dangerous in general, it will make it easy for programmers to make incorrect assumptions about the cluster state.

e.g., kube-proxy may lag behind apiserver by many versions, so if you assume your local value of the feature state matches the actual cluster value, you're going to have a bad time.